lib/shared/config/customize.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var config = require('./');
+var merge = require('lodash.merge');
+var isString = require('lodash.isstring');
+
+// If project dir and cwd are same, the gulprc file in the directory is loaded
+// twice continuously. This function is to avoid it.
+var lastLoadedFile = null;
+function isNotLastLoaded(file) {
+  if (lastLoadedFile === file) {
+    return false;
+  }
+  lastLoadedFile = file;
+  return true;
+}
+
+module.exports = function(configFiles) {
+  configFiles.filter(isString).filter(isNotLastLoaded).forEach(function(file) {
+    merge(config, require(file));
```

Comment:
I think merging with `opts` is good! First I thought the config appears anywhere in gulp-cli and customizes its behaviors directly. But if the `opts` is merged with the config (or a new object is merged with the `opts` and the config) and gulp-cli uses only the `opts` (or the new object) for configuring its behaviors, it seems neater.

And I think this merging should be done at the top of `handleArguments` in `index.js`.